common/lib/xmodule/xmodule/js/src/sequence/display.coffee

```diff
@@ -71,22 +71,22 @@ class @Sequence
         when 'done' then element.addClass('progress-done')
 
   toggleArrows: =>
-    @$('.sequence-nav-buttons a').unbind('click')
+    @$('.sequence-nav-button').unbind('click')
 
     if @contents.length == 0
-      @$('.sequence-nav-buttons .prev a').addClass('disabled').attr('aria-hidden', 'true')
-      @$('.sequence-nav-buttons .next a').addClass('disabled').attr('aria-hidden', 'true')
+      @$('.sequence-nav-button.button-previous').addClass('disabled').attr('aria-hidden', 'true').attr('disabled', 'disabled')
```

Comment:
Ah good find. Wouldn't the normal `disabled` attribute work just the same, or do we need `aria-disabled`?